olm-catalog/serverless-operator/project.yaml

```diff
@@ -5,8 +5,8 @@ project:
   version: 1.28.0
 
 olm:
-  replaces: 1.27.0
-  skipRange: '>=1.27.0 <1.28.0'
+  replaces: 1.27.1
+  skipRange: '>=1.27.1 <1.28.0'
```

Comment:
I believe the skipRange should be kept >=1.27.0 , as we still want users on 1.27.0 to go directly to 1.28.0?

Reply:
I'm a little confused with the upgrade tree, yeah you are correct this will allow both (1.27.0 & 1.27.1) the user to update to 1.28.0, but I have a dought let's say the user with 1.27.0 looking to update the to the 1.28.0 & here we replace 1.27.1 is it okay for them or they just need to go to 1.27.1 and then 1.28?

Reply:
From the functionality point of view, it should work fine if 1.27.1 is skipped when upgrading from 1.27.0 (1.27.1 is just a rebuild with security updates, so there is no migration code or anything different between 1.27.0 and 1.27.1... upgrading from 1.27.0 to 1.27.1  is therefore functionally not different to just reinstalling 1.27.0 , so, skipping 1.27.1 should be fine)